### ISSUE TITLE
Whole Editor UI Broken

### ISSUE THREAD

**Issue by anotherREYY** [godotengine/godot#35971]
**Godot version : **
3.2 stable

**OS/device including version :**
x11 (Debian 10) 32 bit with 1080p monitor

**Issue description :**
first time i opened the editor was pretty lag , saving the setting or switching to different window would instantly freeze the software .
then today i opened the editor seems corrupt , text is missing , everything is misaligned and the popup is invisible , deleting the config file does nothing

**Screenshots :**
(on actual resolution)
![Screenshot](https://user-images.githubusercontent.com/52782833/73999037-12f21b00-4996-11ea-9c81-c2800af8e1ec.png)

**Comment by groud** [comment]
Looks like a driver bug, not sure we can do something about it.

**Comment by anotherREYY** [comment]
Hm , so v3.2 did some major changes to the driver compatibility ? Because befo

**Comment by anotherREYY** [comment]
So v3.2 did a major changes to the driver compatibility ? 

Because back then when I run v3.1 it runs smoothly , the problem occurs after I use v3.2 . strangely when I tries to open the 3.1 again , it act same as v3.2 , showing the same weird ui corruption things.

Then I thought maybe it was because of the config folder , but nothing changed after I delete .config/godot or .local/share/godot

**Comment by akien-mga** [comment]
> So v3.2 did a major changes to the driver compatibility ?
>
> Because back then when I run v3.1 it runs smoothly , the problem occurs after I use v3.2 . strangely when I tries to open the 3.1 again , it act same as v3.2 , showing the same weird ui corruption things.

If you have the same issue with 3.1 and 3.2, then the problem is not linked to 3.2 but indeed to your system.

You can try to update your system, reboot to be sure to pick up kernel updates, and see if it fixes it.

Does it happen in a new project or only a specific one?

**Comment by groud** [comment]
Maybe it could be due to corrupted editor or project settings ?

**Comment by Anutrix** [comment]
Maybe your PC drivers co-incidentally updated/switched while you moved to 3.2. Check if your drivers updated recently.

**Comment by SoapSpangledGames** [comment]
I'm having a similar problem, but only with a specific project.  Although I have recently upgraded my system, which included an nvidia driver upgrade, it appears to be a corrupt configuration, since the entire user interface becomes unusable for this one project, but my others are fine.

Here is what I think I did to invoke this problem on my system:

1) The project started as a Godot 3.1 project.
2) I compiled a custom 3.2 alpha, and worked on this project (and only this project, I think).
3) When 3.2 stable was released, I used it for this project (and only this project, I think).
4) I later accidentally loaded this project (and only this project, I think) into my 3.2-alpha version.  This caused the entire IDE to freeze and spike my CPU to a very high load (I don't remember the specific load).
5) I killed the 3.2 alpha, and reloaded into into 3.2 stable.  I had lots of trouble getting the project to load and run, but it ran after many attempts of loading/killing Godot.  At this point, the IDE exhibited lots of different problems with just this project.

I've posted a 31-second video on my YouTube channel showing the problem at:

https://www.youtube.com/watch?v=JvX3N3wA6eM

Here are some (possibly) relevant system details the inxi utility:

Desktop: KDE Plasma 5.12.9 (Qt 5.9.5) Distro: Ubuntu 18.04.4 LTS
Card: NVIDIA GP108 [GeForce GT 1030]

What the video doesn't show (for lack of time) is how even the text in the debug panel eventually (but not immediately) becomes solid rectangular white blocks where each character should be.

I suspect it's a corrupt configuration, but I don't know where to start looking.  I'm posting a screenshot of the initial corrupted GUI in case you don't want to look at the video.

![image](https://user-images.githubusercontent.com/55257920/75151799-265af100-56cd-11ea-855a-450012989557.png)

**Comment by SoapSpangledGames** [comment]
I fixed my own issue by exiting Godot, deleting ~/.config/godot/projects, and then restarting Godot and reloading my project.  Not only is the editor UI working correctly again, but my project loads nearly instantaneously again (something I hadn't realized had become slow).

I've posted another short video on my YouTube channel demonstrating Godot's behavior after deleting the ~/.config/Godot/projects folder:

https://www.youtube.com/watch?v=NR_vevS-d0Q

Here a screenshot showing the corrected layout.

![image](https://user-images.githubusercontent.com/55257920/75243901-60d99200-5790-11ea-86c2-ce460650594b.png)
